src/test/java/utils/FileHelper.java

```diff
@@ -35,4 +43,12 @@ public static String readJsonFileAsString(String path) {
       throw new TestExecutionException(ioException.getMessage());
     }
   }
+
+  public static List<Product> readFileAsProduct(String path) {
```

Comment:
First of all, try to generify this method like "readJsonFileAsObject". Also, please provide a more meaningful name for the method.